### PR TITLE
DVC-930 add ISO-639-1 validator

### PR DESCRIPTION
**Pull request by squarepegg**

### CODE REVIEW

File: lib/shared/types/src/types/apis/sdk/clientSDKAPI.ts
Comment (ajwootto) [inline lib/shared/types/src/types/apis/sdk/clientSDKAPI.ts:205]: should we be collecting the language automatically in the SDKs like we did in TL, or leave it up to the user?

Comment (squarepegg) [reply lib/shared/types/src/types/apis/sdk/clientSDKAPI.ts:202]: A good discussion point, but a far different change than addressing an outstanding issue.

Happy to put this on hold until this we can bring this up at the next sdk guild.

**Review by ajwootto — APPROVED** [review]

**Pull request merged**
